spec/requests/admin/configs_spec.rb

```diff
@@ -79,6 +79,14 @@
           expect(SiteConfig.authentication_providers).to eq([provider])
         end
 
+        it "allows all providers to be disabled" do
+          # We have to send _something_ for an update, so we'll simulate the borwser omitting the auth
+          # providers array when no options are selected
+          post "/admin/config", params: { site_config: { community_description: "test" },
```

Comment:
This test might need to be updated, I think I added a request spec for this in my PR, so we might be able to just drop this one.

There isn't a system spec for it, which is the real issue that this change is addressing, as I understand it.

Reply:
So the thing about unselecting all auth providers in the UI is that the browser won't send that key to the backend: the `authentication_providers` key will simply be omitted from params. Any request test that has `authentication_providers: []` as a param is constructing a situation that can't occur!

I tried constructing a test with a blank `site_config` hash, but since it's a require parameter Rails won't let that fly. I've changed this test to have a just `ga_tracking_id`